pkg/scheduler/server/internal/cron/cron.go

```diff
@@ -108,28 +120,70 @@ func (c *cron) Run(ctx context.Context) error {
 		return err
 	}
 
+	ch := make(chan []*anypb.Any)
```

Comment:
nit for readability:
```suggestion
	WatchLeadershipCh := make(chan []*anypb.Any)
```